File: Kritika Tripathi/Basics Of Programming/Getting Started/count_digits.cpp
#include <iostream>
using namespace std;

int main(){
    int n;
    cin >> n;
    
    //write your code here
    int d = 0, temp = n;
    while(temp!=0){
        d++;
        temp/=10;
    }
    cout<<d<<endl;
    
}

File: V.Vennela/Basics of Programming/CountDigits.java
import java.util.*;
  
  public class CountDigits{
  
  public static void main(String[] args) {
    // write your code here  
    int count=0;
    Scanner sc = new Scanner(System.in);
    int n = sc.nextInt();
    while(n!=0)
    {
        count++;
        n=n/10;
    }
    System.out.println(count);
   }
  }
